Can you please create a new release with scoverage disabled

Latest release has: "coverageEnabled := true"

I suspect this is causing something to be left behind in the latest release as I'm getting the following error when trying to use the client library:

Caused by: java.io.FileNotFoundException: /Users/rcschermesser/Projects/algoliasearch-client-scala/target/scala-2.11/scoverage-data/scoverage.measurements.147 (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileWriter.<init>(FileWriter.java:107)
	at scoverage.Invoker$$anonfun$1.apply(Invoker.scala:42)
	at scoverage.Invoker$$anonfun$1.apply(Invoker.scala:42)
	at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:901)
	at scoverage.Invoker$.invoked(Invoker.scala:42)
	at algolia.AlgoliaClientConfiguration$.<init>(AlgoliaClientConfiguration.scala:37)
	at algolia.AlgoliaClientConfiguration$.<clinit>(AlgoliaClientConfiguration.scala)

--- a/build.sbt
+++ b/build.sbt
@@ -10,7 +10,7 @@ crossScalaVersions := Seq("2.11.11", "2.12.2")
 
 scalaVersion := "2.12.2"
 
-coverageEnabled := true
+coverageEnabled := false
 
 val asyncHttpClientVersion = "2.0.33"
 val json4sVersion = "3.5.2"

